### ISSUE TITLE
formating text in schedule (before arrival)

### ISSUE THREAD

**Issue by msalganik** [compsocialscience/summer-institute#11]
@lukebaker 

I've tried to update the schedule to show what students should do before arrival.
https://github.com/compsocialscience/summer-institute/commit/1bace7df3fb48c0674e8c4cc629c706916b041e3

Then my attempted fix did not compile:
https://github.com/compsocialscience/summer-institute/commit/b77dabdea6c002921423e8aa938cc581caff2fe6

Basically, the text is not formatting the way that I expected.  Do you have any suggestions?

I know that you are away on now, and this can wait until you get back.

**Comment by lukebaker** [comment]
Matt, I've fixed this in c278936fa2ca6f3c4718d0cfcd8a3b1ef011eca4.

For the YAML formatting, when you want to have a string on multiple lines and keep the newlines that are included in the string, then you should use `|`. We want to keep the newlines because you want this string to be interpreted as markdown, which expects newlines for delineating paragraphs. For example:

```yaml
- date: Before arrival
  events:
    - name: |
        Here's paragraph 1.

        Here's paragraph 2.
```

I also needed to update the HTML template to allow the event name to be parsed as markdown. Finally, I updated the bare links to be wrapped in the markdown link notation.

You can see it working here: https://compsocialscience.github.io/summer-institute/2017/#schedule

**Comment by msalganik** [comment]
Great.  Thanks @lukebaker
